src/controllers/projectController.ts

```diff
@@ -53,12 +53,11 @@ export class ProjectController {
         const projectFile: string = path.join(basePath, projectName, ".project");
         const extensionPath: string = this.context.extensionPath;
         try {
-            await Promise.all([
-                fse.copy(path.join(extensionPath, "templates", "App.java.sample"), path.join(basePath, projectName, "src", "app", "App.java")),
-                fse.copy(path.join(extensionPath, "templates", `Java${javaVersion}`), path.join(basePath, projectName)),
-                fse.copy(path.join(extensionPath, "templates", ".project"), projectFile),
-                fse.ensureDir(path.join(basePath, projectName, "bin")),
-            ]);
+            await fse.ensureDir(path.join(basePath, projectName, "bin"));
```

Comment:
These two has very different meanings. The later makes it a serial operation. I don't see a problem with Promise.all. Please keep that one.